File: Python/1149.py
x = list(map(int, input().split()))
i = 1
s = 0

while x[i] <= 0:
    if x[i] <=0:
        i = i + 1
    if x[i] > 0:
        break

for c in range(0,x[i]):
    s = s + x[0] + c
    
print(s)

File: CSharp/1149.cs
using System;
using System.Linq;

class URI
{
    static void Main()
    {
       int[] x = Console.ReadLine().Split().Select(int.Parse).ToArray();
       int a = x[0];
       int n=0;
       int soma=0;

       for(int i =1;i<x.Length;i++)
       {
            if(x[i]>0)
            {n =x[i];break;}
       }
       for(int i=0;i<n;i++)
       {
            soma+=a+i;
       }
       Console.WriteLine(soma);
    }
}